en/docs/api-analytics/getting-started-guide.md

```diff
@@ -8,7 +8,7 @@ Follow the instructions below to sign in to Choreo.
 
 1. Navigate to Choreo using the following URL. 
     
-     [https://console.choreo.dev/?apianalytics=true](https://console.choreo.dev/?apianalytics=true).
+     [https://console.choreo.dev/?apianalytics=true](https://console.choreo.dev/?apianalytics=true?utm_source=apim_docs).
```

Comment:
shall we change this to 
[https://console.choreo.dev](https://console.choreo.dev?apianalytics=true&utm_source=apim_docs).